.github/workflows/deploy-book-ghpages.yml

```diff
@@ -72,6 +72,12 @@ jobs:
     - if: ${{ steps.cache-html.outputs.cache-hit != 'true' }}
       name: Preprocess & build the book from branch
       run: |
+        echo $PATH
+        pip install --user teachbooks --extra-index-url https://gitlab.tudelft.nl/api/v4/projects/13957/packages/pypi/simple
```

Comment:
Will need to update this line once PyPI is set up.

Reply:
Depends on https://github.com/TeachBooks/TeachBooks/issues/1